src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1542,6 +1542,27 @@ private void visitTypeDeclaration(JSType type, boolean isVarArgs, boolean isOpti
       }
     }
 
+    private boolean emitNoResolvedType(NoType type, String defaultEmit) {
```

Comment:
add some docs on what this does? It's not obvious from the signature.

Reply:
done